packages/converters/src/boolean-to-number/index.test.ts

```diff
@@ -13,15 +13,12 @@
 import { describe, expect, it } from 'vitest';
 import { booleanToNumber } from './';
 
-const testPairs: [boolean, number][] = [
-  [true, 1],
-  [false, 0],
-];
-
 describe('bool to number', () => {
-  for (const [input, result] of testPairs) {
-    it(`should convert ${input} to ${result}`, () => {
-      expect(booleanToNumber(input)).toEqual(result);
-    });
-  }
+  it.each`
+    input    | result
+    ${true}  | ${1}
+    ${false} | ${0}
+  `('should convert $input to $result', ({ input, result }) => {
+    expect(booleanToNumber(input)).toEqual(result);
```

Comment:
Prefer not to use string template: https://github.com/gohypergiant/standard-toolkit/pull/88#discussion_r1834744892

Reply:
That's subjective. Array, or object, allows much more flexibility and less typing (in the case of array).